src/Parser/Hunter/BeastMastery/Modules/Features/MarkOfTheClaw.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import { formatPercentage } from "common/format";
+
+class MarkOfTheClaw extends Analyzer {
```

Comment:
Please reconsider this module. Mark of the Claw makes a very small contribution to damage, the uptime is completely outside of the player's control, and the enchant is not BeastMastery specific so I don't know why it's in the BeastMastery directory.

Reply:
Fair enough I guess

Reply:
It might be reasonable in the future to include throughput or "average stat gain" calculations for each of the neck enchants, but I don't think including a full sized statistic that doesn't expose any interesting information is valuable.